tests/unittests/test_packets_static.c

```diff
@@ -271,6 +271,151 @@ static void test_rtr_pdu_check_size(void **state)
 	assert_false(rtr_pdu_check_size((struct pdu_header *)error));
 }
 
+static void test_rtr_pdu_check_interval(void **state)
+{
+	UNUSED(state);
+
+	struct rtr_socket *rtr_socket = malloc(1024);
+	struct pdu_end_of_data_v1 *pdu_eod = malloc(1024);
+
+	int retval;
+
+	rtr_socket->refresh_interval = 0;
+	rtr_socket->retry_interval = 0;
+	rtr_socket->expire_interval = 0;
+
+	pdu_eod->refresh_interval = 1;
+	pdu_eod->retry_interval = 2;
+	pdu_eod->expire_interval = 601;
+
+	/* test appliance of interval values to the rtr_socket */
+	apply_interval_value(rtr_socket, pdu_eod->refresh_interval, REFRESH);
+	assert_int_equal(rtr_socket->refresh_interval,
+			 pdu_eod->refresh_interval);
+
+	apply_interval_value(rtr_socket, pdu_eod->retry_interval, RETRY);
+	assert_int_equal(rtr_socket->retry_interval, pdu_eod->retry_interval);
+
+	apply_interval_value(rtr_socket, pdu_eod->expire_interval, EXPIRATION);
+	assert_int_equal(rtr_socket->expire_interval, pdu_eod->expire_interval);
+
+	/* test checks that determine if value is inside range */
+	retval = 2;
```

Comment:
why is `retval` set here, or below, when it is replaced by function return value afterwards?